src/collections/_documentation/platforms/node/serverless.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Serverless
+sidebar_order: 7
+---
+
+## AWS Lambda
+
+To set up Sentry error logging for a Lambda function, build a wrapper.
+
+```javascript
+    "use strict";
+
+    const Sentry = require("@sentry/node");
+
+    Sentry.init({
+    dsn: "YOUR DSN",
+    });
+
+    function sentryHandler(lambdaHandler) {
+    return async event => {
+        try {
+        return await lambdaHandler(event);
+        } catch (error) {
+        Sentry.captureException(error);
+        await Sentry.flush(2000);
+        return {
+            body: JSON.stringify({
+            err: error.message,
```

Comment:
According to the docs https://docs.aws.amazon.com/lambda/latest/dg/nodejs-prog-mode-exceptions.html it should be `${error. errorType}: ${error.errorMessage}`

Reply:
Oh nice. How would you revise the code snippet?

Reply:
`return error;` :)

Reply:
Lol. 😛 Updated.